AmericanExpress/src/main/java/com/braintreepayments/api/AmericanExpressClient.java

```diff
@@ -36,6 +36,7 @@ public void getRewardsBalance(@NonNull String nonce, @NonNull String currencyIso
                 .appendQueryParameter("currencyIsoCode", currencyIsoCode)
                 .build()
                 .toString();
+        String analytis = AmericanExpressAnalytics.REWARDS_BALANCE_SUCCEEDED.event;
```

Comment:
Accidental commit?

```suggestion
```

Reply:
Ah yes whoops was testing to confirm the Kotlin conversion worked correctly!